README.md

```diff
@@ -12,6 +12,7 @@ For more information: https://hackmd.io/@Vid201/aa-bundler-rust
 
 ## Prerequisites
 
+1. Ethereum JSON-RPC API with enabled [`debug_traceCall`](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-debug#debug_tracecall) (currently implemented only in [Geth](https://github.com/ethereum/go-ethereum) and [Erigon](https://github.com/ledgerwatch/erigon)). For testing purposes, you can setup [private Geth node](https://github.com/krzkaczor/geth-private-node).  
```

Comment:
I am just curious what kind of situation you need `debug_traceCall`.

Reply:
`debug_traceCall` is needed to analyze the user operation simulation and its opcodes (https://github.com/eth-infinitism/account-abstraction/blob/develop/eip/EIPS/eip-4337.md#specification-1).

Here is an example of the other bundler implementation (eth-infinitism): 
- https://github.com/eth-infinitism/bundler/blob/main/packages/bundler/src/GethTracer.ts#L18
- https://github.com/eth-infinitism/bundler/blob/270ff689877c20e58b396e5b092ee07a67510751/packages/bundler/src/modules/ValidationManager.ts#L122

Reply:
@zsluedem

Reply:
Sorry for the late approve vid. I wasn't feel well yesterday.

Reply:
No problem, man; I hope you are feeling better now.

I pinged you again in case you missed the comment.

Reply:
Thanks! . I get it now~! I was wondering whether the API could be replaced by others because debug_tracecall is not standard exectution api. But I could not find another.